### ISSUE TITLE
Error using assert Formatted arguments must be scalar.

### ISSUE THREAD

**Issue by JohnG0024** [alecjacobson/gptoolbox#104]
Hi, I keep getting this error when trying to run the `lscm` function. How can I fix it ?
 
```
Error using assert
Formatted arguments must be scalar.

Error in min_quad_with_fixed/precompute (line 134)
    assert(isempty(known) || min(size(known))==1, ...

Error in min_quad_with_fixed (line 101)
    F = precompute(A,known,Aeq,F);

Error in lscm (line 126)
    U = min_quad_with_fixed(Q,zeros(2*n,1),[b b+n],bc(:),Aeq,Beq);

Error in testTriangulationMatcher (line 94)
[U, Q ] = lscm(Vu, T, hh, hl); % Vu : nV x 3, T : nF x 3, hh : boundary_id list, hl : length(hh) x 2 
```

**Comment by JohnG0024** [comment]
Test example. [test_lscm.zip](https://github.com/alecjacobson/gptoolbox/files/5607522/test_lscm.zip)
Appreciate it.

**Comment by alecjacobson** [comment]
Should be fixed in 4d103056f46ae3ab2d2ed84612692afe5bb6e036 so if you `git pull`
